nautobot/project-static/css/base.css

```diff
@@ -951,6 +951,17 @@ code.hljs {
     flex-direction: column;
     padding: 20px;
 }
+.tile.clickable {
```

Comment:
Do the changes in this file have any negative impact on the Jobs tile view?

Reply:
nope, I made double sure any changes to `base.css` are safe for all existing pages